api/modules/feedback/views.py

```diff
@@ -39,15 +39,11 @@ def get_feedback(request, feedback_id):
     Returns the feedback pertaining to a certain feedback id
```

Comment:
Checkout untouched files

Reply:
what do i do in this. i didnt touch this file

Reply:
This might help you https://stackoverflow.com/a/28173964/5602759

Reply:
as you can see the file was not untouched and some changes were  made.